pkgs/servers/matrix-synapse/plugins/s3-storage-provider.nix

```diff
@@ -16,13 +30,26 @@ buildPythonPackage rec {
       --replace "humanize>=0.5.1,<0.6" "humanize>=0.5.1"
   '';
 
+  buildInputs = [
+    matrix-synapse-unwrapped
+  ];
+
+  propagatedBuildInputs = [
+    boto3
+    humanize
+    psycopg2
```

Comment:
psycopg2 etc. are implied in `matrix-synapse-unwrapped.propagatedBuildInputs`. I would prefer if this stayed in sync with the exact inputs used by synapse

Reply:
Makes no sense to do it different with this plugin, change reverted.